src/cmd-build

```diff
@@ -25,11 +25,12 @@ EOF
 
 # Parse options
 FORCE=
+FORCE_BUILD=
```

Comment:
Why not `FORCE_IMAGE`?

Reply:
yes i guess FORCE_IMAGE is more appropriate. will change it to FORCE_IMAGE